packages/react-mutation-mapper/src/component/dataTable/DataTable.tsx

```diff
@@ -162,17 +162,6 @@ export default class DataTable<T> extends React.Component<
         );
     }
 
-    @computed
-    get defaultPageSize() {
-        const initialItemsPerPage = this.props.initialItemsPerPage;
-
-        return this.tableData
-            ? this.tableData.length > initialItemsPerPage!
-                ? initialItemsPerPage
-                : this.tableData.length
-            : 1;
-    }
```

Comment:
@onursumer how come we can get rid of this logic now?

Reply:
@inodb Apparently this wasn't necessary in the first place. I thought this was required to get rid of the empty padding rows, but actually setting `minRows` to a small number is good enough. See the `minRows` property here: https://github.com/tannerlinsley/react-table/tree/v6#props.

Setting `defaultPageSize` to a computed value, especially to zero, messes things for the pagination.